pkgs/applications/misc/blender/default.nix

```diff
@@ -58,13 +57,10 @@ stdenv.mkDerivation rec {
       "-DPYTHON_VERSION=${python.pythonVersion}"
       "-DWITH_PYTHON_INSTALL=OFF"
       "-DWITH_PYTHON_INSTALL_NUMPY=OFF"
+      "-DPYTHON_NUMPY_PATH=${pythonPackages.numpy}/${python.sitePackages}"
     ]
     ++ optional jackaudioSupport "-DWITH_JACK=ON"
-    ++ optionals cudaSupport
-      [ "-DWITH_CYCLES_CUDA_BINARIES=ON"
-        # Disable architectures before sm_30 to support new CUDA toolkits.
-        "-DCYCLES_CUDA_BINARIES_ARCH=sm_30;sm_35;sm_37;sm_50;sm_52;sm_60;sm_61"
```

Comment:
Seems like it isn't needed anymore [1].

[1] https://git.blender.org/gitweb/gitweb.cgi/blender.git/blob/v2.80:/CMakeLists.txt#l410